exchanges/multipart-fetch/src/multipartFetchExchange.test.ts

```diff
@@ -153,7 +153,7 @@ describe('on error', () => {
     expect(data).toMatchSnapshot();
   });
 
-  it('ignores the error when a result is available', async () => {
+  it.skip('ignores the error when a result is available', async () => {
```

Comment:
@JoviDeCroock @kitten — can you please assist me. Was struggling to understand the point of this test? If there was an error (400), then we should error and not return data?

Or is the idea to always return the data — even if there was an error?

Reply:
We allow for errors and results to both be in the response, think about a sub-resolver failing, that sub-resolver will insert an error message into the [`CombinedError`](https://formidable.com/open-source/urql/docs/basics/errors/) but when that sub-resolver is optional we can still return a valid result to the user.

Reply:
Should be ready again for you @JoviDeCroock 🤘🏻